tests/from_meta.rs

```diff
@@ -64,3 +64,55 @@ fn nested_meta_lit_bool_errors() {
         Error::unsupported_format("literal").to_string()
     );
 }
+
+/// Tests behavior of FromMeta implementation for enums (including behavior of defaults).
+mod enum_impl {
+    use darling::{Error, FromMeta};
+    use syn::parse_quote;
+
+    /// A playback volume.
+    /// Deriving `Default` will cause the "default" variant to be set when the meta-item is empty.
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, Default, FromMeta)]
```

Comment:
It looks like deriving `Default` on the enum breaks 1.56, which is part of Rust's MSRV. I don't think adding a derived `Default` (as opposed to an `impl` alters this case meaningfully, so I'd suggest adding a new `tests/enums_default.rs` folder and then having that showcase the two relevant behaviors:

1. Using `#[darling(default)]` on the receiver field
2. Using `#[darling(word)]` on the variant that is used when only a word is present.

```rust
use darling::{FromDeriveInput, FromMeta};
use syn::parse_quote;

#[derive(Debug, FromMeta, PartialEq, Eq)]
enum Dolor {
    Sit,
    #[darling(word)]
    Amet,
}

impl Default for Dolor {
    fn default() -> Self {
        Dolor::Sit
    }
}

#[derive(FromDeriveInput)]
#[darling(attributes(hello))]
struct Receiver {
    #[darling(default)]
    example: Dolor,
}

#[test]
fn missing_meta() {
    let di = Receiver::from_derive_input(&parse_quote! {
        #[hello]
        struct Example;
    })
    .unwrap();

    assert_eq!(Dolor::Sit, di.example);
}

#[test]
fn empty_meta() {
    let di = Receiver::from_derive_input(&parse_quote! {
        #[hello(example)]
        struct Example;
    })
    .unwrap();

    assert_eq!(Dolor::Amet, di.example);
}
```

Reply:
Will switch to an `impl` for `Default`.

Also I get this error when trying the above code snippet:
```
error: Unknown field: `word`
 --> tests/enums_default.rs:7:15
  |
7 |     #[darling(word)]
  |               ^^^^
```

Reply:
Makes sense that you'd get an error, since `#[darling(word)]` isn't implemented - that's the alternative syntax to `#[default]` that I proposed in another thread in this PR.

Reply:
I guess it was a bit of confusion on both our parts. 
The `#[default]` syntax in my example is actually from <https://doc.rust-lang.org/std/default/trait.Default.html#enums> so that's not a custom attribute I was adding.